_posts/en/newsletters/2024-01-24-newsletter.md

```diff
@@ -0,0 +1,233 @@
+---
+title: 'Bitcoin Optech Newsletter #286'
+permalink: /en/newsletters/2024/01/24/
+name: 2024-01-24-newsletter
+slug: 2024-01-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Disclosure of fixed consensus failure in btcd:** "Niklas Gögge
+  used Delving Bitcoin to [announce][gogge btcd] a consensus failure in
+  older versions of btcd that he had previously [responsibly
+  disclosed][topic responsible disclosures].  Relative time locks were
+  added to Bitcoin in a soft fork by adding consensus-enforced meaning
+  to transaction input sequence numbers.  To ensure any presigned
+  transactions created before the fork would not be made invalid, the
+  relative time locks only apply to transactions with version numbers 2
+  and higher, allowing transactions with the original default of version
+  1 to remain valid with any inputs.  However, in the original Bitcoin
+  software, version numbers are signed integers, meaning negative
+  versions are possible.  The _reference implementation_ section of
+  [BIP68][] notes that "version 2 and
+  higher" is meant to apply to the version number cast from a signed
+  integer into an unsigned integer, ensuring that the rules apply to any
+  transaction that isn't version 0 or 1.
+
+    Gögge discovered that btcd did not implement this conversion from
```

Comment:
not sure but maybe here and elsewhere:
```suggestion
    Gögge discovered that `btcd` did not implement this conversion from
```

Reply:
btcd is the name of the software, so backticks definitely aren't appropriate.